site/.eleventy.js

```diff
@@ -0,0 +1,181 @@
+const historyFiltering = require("./src/tools/history_filter")
+const calculate = require("./src/tools/calculate")
+
+const friendlyBoardName = {
+  "intel-nuc": "Intel NUC",
+  "generic-aarch64": "Generic AArch64",
+  "generic-x86-64": "Generic x86-64",
+  "khadas-vim3": "Khadas VIM3",
+  ova: "Virtual Machine",
+  "odroid-c2": "ODROID-C2",
+  "odroid-c4": "ODROID-C4",
+  "odroid-n2": "ODROID-N2",
+  "odroid-xu4": "ODROID-XU4",
+  rpi: "Raspberry Pi",
+  rpi0: "Raspberry Pi Zero",
+  "rpi0-w": "Raspberry Pi Zero W",
+  rpi2: "Raspberry Pi 2",
+  rpi3: "Raspberry Pi 3 (32-bit)",
+  "rpi3-64": "Raspberry Pi 3",
+  rpi4: "Raspberry Pi 4 (32-bit)",
+  "rpi4-64": "Raspberry Pi 4",
+  tinker: "ASUS Tinker Board",
+  yellow: "Home Assistant Yellow",
+};
+
+const randomColor = () => `#${Math.floor(Math.random() * 16777215).toString(16)}`
```

Comment:
it would be cool if our random color could be based on the name, so it's deterministic and the same across graphs/updates.